lib/language.test.js

```diff
@@ -73,31 +77,33 @@ describe('languageHandler', () => {
 
   it('should use language cookie if requestedLanguage is invalid', () => {
     const requestedLanguage = 'fr'
-    const clientLanguage = 'en'
+    const acceptLanguage = preferredLanguages
+    const expectedLanguage = 'en'
     const req = {
       query: { l: requestedLanguage },
       cookies: {},
-      headers: { 'accept-language': clientLanguage },
+      headers: { 'accept-language': acceptLanguage },
     }
     const res = { locals: {}, cookie: jest.fn() }
     const next = jest.fn()
 
     languageHandler(req, res, next)
-    expect(res.cookie).toHaveBeenCalledWith(cookieName, clientLanguage, cookieOptions)
+    expect(res.cookie).toHaveBeenCalledWith(cookieName, expectedLanguage, cookieOptions)
   })
 
   it('should use client language if cookie language is invalid', () => {
```

Comment:
I think this test is a bit unclear now, why do we expect 'en'? I assume that it is because it's the first in the list of `preferredLanguages`? But that is very hard to tell from just looking at the test